build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     gradlePluginPortal()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.6.1'
+    classpath 'com.android.tools.build:gradle:4.2.2'
```

Comment:
Please revert this change. It should not be part of this PR.

Reply:
Done